.github/ci-scripts/bash/functions-for-build

```diff
@@ -565,6 +591,61 @@ setup_qa_env() {
   deactivate
 }
 
+setup_packer_env() {
+  sudo python3 -m pip uninstall -y packer_framework
+  python3 -m pip install --user virtualenv --trusted-host ${NEXUS_SERVER_ENDPOINT}
+  python3 -m venv pkenv
+  source pkenv/bin/activate
+  python3 -m pip install wheel --trusted-host ${NEXUS_SERVER_ENDPOINT}
+  python3 -m pip install -r ./requirements.txt --extra-index-url=https://$GITHUB_USER:$GITHUB_TOKEN@nexus.tessell.cloud/repository/${NEXUS_PULL_REPOS_PY}/simple
+}
+
+software_def_build() {
+  setupPackerEnv
+  export CURRENT_VERSION_TAG="0.0.0"
+  mkdir -p ~/.packer.d/plugins
+  if [[ "$BRANCH_NAME" == *double_commit* ]]; then
+    echo "BRANCH_NAME contains double_commit"
```

Comment:
Use `"${BRANCH_NAME} contains double_commit"` ; This way it will be helpful in debugging.